### PR TITLE
Fix for setuptools 50.x

### PR DESCRIPTION
**Pull request by ojasjoshi**
*Issue #, if available:*

*Description of changes:*
- Blackisting setuptools v50.0.0
- Currently the blacklisted version for setuptools is overidden during colon-bundle pip installation.
- This change adds the blacklisted installation AFTER colcon-bundle 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: ros_bootstrap.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ set -xe
 
 # Set up ROS APT and install basic dependencies (rosdep, rosinstall). Must have ROS_VERSION set when called.
 sudo apt-get update && sudo apt-get install -q -y dirmngr curl gnupg2 lsb-release zip python3-pip python3-apt dpkg
-sudo -H pip3 install -U setuptools!=50.0.0
+# sudo -H pip3 install -U setuptools!=50.0.0
```
Comment (mm318) [inline ros_bootstrap.sh:6]: When finalizing the PR, let's remove this line instead of commenting it out.

Comment (ahtsan) [reply ros_bootstrap.sh]: Done

**Comment by ojasjoshi** [comment]
Closing this PR for now.. I think [colcon-bundle blacklisting setuptools v50.0.0](https://github.com/colcon/colcon-bundle/pull/179) should solve the issue

**Comment by ahtsan** [comment]
Reopening PR as CI still fails

File: ros_bootstrap.sh [hunk 2]
```diff
@@ -3,7 +3,6 @@ set -xe
 
 # Set up ROS APT and install basic dependencies (rosdep, rosinstall). Must have ROS_VERSION set when called.
 sudo apt-get update && sudo apt-get install -q -y dirmngr curl gnupg2 lsb-release zip python3-pip python3-apt dpkg
-sudo -H pip3 install -U setuptools!=50.0.0
```
Comment (ojasjoshi) [inline ros_bootstrap.sh:6]: I think let's keep this with <50

File: ros1_sa_build.sh [hunk 3]
```diff
@@ -8,6 +8,8 @@ ROS_BOOTSTRAP_SCRIPT=${SCRIPT_DIR}/ros_bootstrap.sh
 "${ROS_BOOTSTRAP_SCRIPT}"
 sudo apt-get update && sudo apt-get install --no-install-recommends -y python3-colcon-common-extensions ros-${ROS_DISTRO}-ros-base
 sudo -H pip3 install -U colcon-bundle colcon-ros-bundle
+# colcon-bundle setup requirements reinstalls the latest version(50.0.0) of setuptools.
+sudo -H pip3 install -U setuptools<50.0.0 --force-reinstall
```
Comment (ojasjoshi) [inline ros1_sa_build.sh:12]: Let's not put the setuptools install here, now that colcon-bundle will be updated with <50

**Pull request closed without merging**